config/crd/kustomization.yaml

```diff
@@ -3,6 +3,7 @@
 # It should be run by config/default
 commonLabels:
     cluster.x-k8s.io/v1alpha3: v1alpha3  
+    cluster.x-k8s.io/v1alpha4: v1alpha3  
```

Comment:
This tells upstream CAPI that v1alpha3 of bootstrap resources can be used with v1alpha4 CAPI resources.